app/Http/Controllers/ReportsController.php

```diff
@@ -545,6 +545,10 @@ public function postCustom(Request $request)
                 $header[] = trans('admin/hardware/table.checkout_date');
             }
 
+            if ($request->filled('checkin_date')) {
+                $header[] = trans('general.last') . ' ' . trans('admin/hardware/table.checkin_date');
```

Comment:
We probably don't want to split this here, since it may not translate properly in other languages.

Reply:
That makes sense. Fixed.